packages/now-build-utils/src/detect-builders.ts

```diff
@@ -18,7 +18,7 @@ const config: Config = { zeroConfig: true };
 const MISSING_BUILD_SCRIPT_ERROR: ErrorResponse = {
   code: 'missing_build_script',
   message:
-    'Your `package.json` file is missing a `build` property inside the `script` property.' +
+    'Your `package.json` file is missing a `build` property inside the `scripts` property.' +
```

Comment:
Having "property" in there twice sounds weird to me.

```suggestion
    'Your `package.json` file is missing a `build` command inside the `scripts` object.' +
```